src/middlewares/isGroupOwner.js

```diff
@@ -0,0 +1,10 @@
+import { AuthenticationError } from 'apollo-server-express'
+
+export default (controller) => async (obj, args, context) => {
+  if (!context.currentUser) throw new AuthenticationError('Must be authenticated')
+
+  const group = await context.models.group.findByPk(args.id)
+  if (!group || group.sellerId !== context.currentUser.id) throw new AuthenticationError('Must be group seller')
+
+  return controller(obj, args, context)
+}
```

Comment:
Please take a look at the following link. It explains how to use `combineResolvers` to apply authentication in GraphQL resolvers: https://www.apollographql.com/docs/graphql-tools/resolvers/#graphql-resolvers